Cargo.toml

```diff
@@ -1,26 +1,31 @@
 [package]
 name = "txmodems"
 readme = "README.md"
-repository = "https://github.com/Cosmo-CoDiOS/txmodems"
+repository = "https://github.com/inApril-AS/txmodems"
 description = "Rust library for various MODEM file transfer protocols"
 categories = ["embedded"]
-keywords = ["data-transfer", "MODEM", "file-transfer", "embedded"]
-version = "0.1.3"
-authors = ["Dom Rodriguez <shymega@shymega.org.uk>"]
+keywords = ["serial", "xmodem", "ymodem", "data-transfer", "MODEM", "file-transfer", "embedded"]
+version = "0.2.0"
+authors = [
+	"Dom Rodriguez <shymega@shymega.org.uk>",
+	"Eliud de León <eliud.deleon.10@gmail.com>",
+	"Eskild Jonatan Krumsvik Aanensen <eskild.krumsvik.aanensen@inapril.com>",
+]
```

Comment:
Version bump is fine, and keywords are fine - but you should be asking before adding yourselves to the authors key. I was going to put 'The Cosmo-CoDiOS Developers' at some point, given this project is under that umbrella.